### PR TITLE
feat(ripple): Annotate mdc-ripple for closure

### LINKED ISSUE #341: Annotate mdc-ripple for closure

### PR DESCRIPTION
**Pull request by nckh**
Fixes #341

### CODE REVIEW

File: packages/mdc-ripple/adapter.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint no-unused-vars: [2, {"args": "none"}] */
+
+/**
+ * Adapter for MDC Ripple.
```
Comment (lynnjepsen) [inline packages/mdc-ripple/adapter.js]: This should say more about what MDCRippleAdapter actually does. Look to this as an example
https://github.com/material-components/material-components-web/blob/master/packages/mdc-menu/simple/adapter.js

Comment (nckh) [inline packages/mdc-ripple/adapter.js]: Thanks Lynn! I shamelessly stole Susie's comment and pushed another commit.

File: packages/mdc-ripple/adapter.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+/* eslint no-unused-vars: [2, {"args": "none"}] */
+
+/**
+ * Adapter for MDC Ripple. Provides an interface for managing
+ * - classes
+ * - dom
+ * - CSS variables
+ * - position
+ * - dimensions
+ * - scroll position
```
Comment (lynnjepsen) [inline packages/mdc-ripple/adapter.js]: I dont think you actually look at scroll position at all, so remove this

Comment (nckh) [inline packages/mdc-ripple/adapter.js]: getWindowPageOffset? Or did I misunderstand?

Comment (lynnjepsen) [inline packages/mdc-ripple/adapter.js]: Oh you're totally right. That does make sense

**Comment by nckh** [comment]
Thanks for the quick turnaround Lynn!

**Pull request merged**